docs/docsite/Makefile.sphinx

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    = -j $(CPUS)
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = sdfsdf
+SOURCEDIR     = .
+BUILDDIR      = _build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile.sphinx
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile.sphinx
```

Comment:
% patterns are a GNU Make extension.  So we'll need to change this.   Maybe we just need to spell out all the possibilities?  I can't recall a portable match-anything Makefile rule.